apisix/cli/schema.lua

```diff
@@ -246,18 +405,225 @@ local config_schema = {
                     }
+                },
+                meta = {
+                    type = "object",
+                    properties = {
+                        lua_shared_dict = {
+                            type = "object",
+                        },
+                    }
+                },
+                stream = {
+                    type = "object",
+                    properties = {
+                        enable_access_log = {
+                            type = "boolean",
+                        },
+                        access_log = {
+                            type = "string",
+                        },
+                        access_log_format = {
+                            type = "string",
+                        },
+                        access_log_format_escape = {
+                            type = "string",
+                        },
+                        lua_shared_dict = {
+                            type = "object",
+                        },
+                    }
+                },
+                main_configuration_snippet = {
+                    type = "string"
+                },
+                http_configuration_snippet = {
+                    type = "string"
+                },
+                http_server_configuration_snippet = {
+                    type = "string"
+                },
+                http_server_location_configuration_snippet = {
+                    type = "string"
+                },
+                http_admin_configuration_snippet = {
+                    type = "string"
+                },
+                http_end_configuration_snippet = {
+                    type = "string"
+                },
+                stream_configuration_snippet = {
+                    type = "string"
+                },
+                http = {
+                    type = "object",
+                    properties = {
+                        enable_access_log = {
+                            type = "boolean"
+                        },
+                        access_log = {
+                            type = "string"
+                        },
+                        access_log_buffer = {
+                            type = "integer"
+                        },
+                        access_log_format = {
+                            type = "string"
+                        },
+                        access_log_format_escape = {
+                            type = "string"
+                        },
+                        keepalive_timeout = {
+                            type = "string"
+                        },
+                        client_header_timeout = {
+                            type = "string"
+                        },
+                        client_body_timeout = {
+                            type = "string"
+                        },
+                        client_max_body_size = {
+                            type = "number"
```

Comment:
```suggestion
                            type = "string"
```

Reply:
Let me try. I originally had it as string but it threw an error.

[`f3ecf05` (#10339)](https://github.com/apache/apisix/pull/10339/commits/f3ecf055e025c1b92abc4a0804a9403e31ea8a68)